_includes/css/agency.css

```diff
@@ -14,6 +14,10 @@ body {
     color: #{{ site.data.template.color.secondary-dark }};
 }
 
+div .container {
```

Comment:
we should probably rename the `div.container` class within each section element (products, services, teams, ...) to `.content-container` or `.section-container` because its a bit ambiguous (there is also a container in the header and I'm not sure if we want to override that one too. then we wouldn't have to override the theme